src/components/POLine/LineLinkedInstances/hooks/useLinkedInstances/useLinkedInstances.js

```diff
@@ -53,7 +53,7 @@ export const useLinkedTitles = line => {
     () => {
       const searchParams = {
         limit: LIMIT_MAX,
-        query: `poLineId=${line.id} and instanceId="" sortby title`,
+        query: `poLineId=${line.id} sortby title`,
```

Comment:
Should `poLineId=${line.id}` also be changed to `poLineId==${line.id}`? It is my understanding from the [CQL docs](https://github.com/folio-org/raml-module-builder/tree/51a5950a83daf5a8aed36526c59225732fcbaad1#cql-relations) that `==` will avoid the substring phrase match for strings.

Reply:
Good point, Zak, thank you!